datalake/src/main/java/com/sequenceiq/datalake/service/sdx/CloudbreakPoller.java

```diff
@@ -91,6 +91,11 @@ public void pollFlowStateByFlowIdUntilComplete(String process, String flowId, Lo
         waitForFlowStateByFlowId(process, flowId, sdxId, pollingConfig, Sets.immutableEnumSet(FINISHED), Sets.immutableEnumSet(FAILED, UNKNOWN));
     }
 
+    public void pollFlowStateBySdxClusterUntilComplete(String process, String flowId, SdxCluster sdxCluster, PollingConfig pollingConfig) {
+        LOGGER.debug("Polling start for flowId: {}", flowId);
```

Comment:
Have you verified that this works with the flow chain you added? Just wondering how you tested this part.